lib/functions/function_plot_measures.R

```diff
@@ -2,94 +2,66 @@
 #'
 #' Creates a line plot of measures over time, with customisable labels and colours.
 #'
-#' @param data A dataframe containing the data to plot.
-#' @param measure_names Strings specifiying the names of measure columns to be plotted.
-#' @param custom_labels Strings specifying the names of legend labels.
-#' @param title A string specifying the title of the plot. Default is NULL.
-#' @param x_label A string specifying the label for the x-axis. Default is NULL.
-#' @param y_label A string specifying the label for the y-axis. Default is NULL.
-#' @param color_label A string specifying the label for the color legend. Default is NULL.
-#' @param value_col The name of the dataframe column which contains the y-axis values. Default is "numerator".
-#' @param measure_col The name of the dataframe column which contains the categorical variable. Default is "measure".
+#' @param data A dataframe containing output from the OpenSAFELY measures framework
+#' @param select_measures List of strings specifiying the names in the measure columns to be plotted.
+#' @param select_interval_date Specify date variable interval_start or interval_end.
+#' @param select_value The name of the column which contains the y-axis values: ratio, numerator, or denominator
+#' @param title A string specifying the title of the plot.
+#' @param x_label A string specifying the label for the x-axis.
+#' @param y_label A string specifying the label for the y-axis.
+#' @param colour_var Column name of the colour variable
+#' @param legend_position Position of the guide/legend of the plot
+#' @param guide_label A string specifying the label for the color/guide legend.
+#' @param guide_nrow Number of rows for the colour/guide
+#' @param facet_wrap Logical, specifying whether to include panels using facet_wrap
+#' @param facet_var Variable name used for creating panels
 #'
 #' @return A ggplot object.
 
 plot_measures <- function(
     data,
-    measure_names,
-    custom_labels = NULL,
-    date_col = "interval_end",
-    value_col = "numerator",
-    measure_col = "measure",
+    select_measures = NULL,
```

Comment:
Will select_measures vary, or will it always be numerator? If so, might as well hardcode it?

Reply:
I think you're right, have removed this in 187b1579907882686856d6696d41f0a73735f526